tests/suite/test_virtual_server_tls.py

```diff
@@ -34,9 +34,9 @@ def assert_ssl_error(virtual_server_setup):
         get_server_certificate_subject(virtual_server_setup.public_endpoint.public_ip,
                                        virtual_server_setup.vs_host,
                                        virtual_server_setup.public_endpoint.port_ssl)
-        pytest.fail("We expected an exception here, but didn't get it. Exiting...")
+        pytest.fail("We expected an SSLError here, but didn't get it or got another error. Exiting...")
```

Comment:
if we get another one, will it get printed so that we understand what happened?

Reply:
yes, pytest will post the exception in the stdout.